main.js

```diff
@@ -0,0 +1,71 @@
+const query = 'mickey';
+
+// Get the user agent language, fall back to English if unknown.
+const language = navigator.language !== null ? navigator.language : 'en';
+// language may contain a dash, e.g., 'en-US':
+// Extract language code, avoiding the unnecessary call to split.
+const langCode = language.indexOf( '-' ) === -1 ? language : language.split( '-' )[ 0 ];
+const search_api = `https://${langCode}.wikipedia.org/w/rest.php/v1/search/title?limit=10&q=`;
+// Basic unordered list.
+const resultList = document.createElement( 'ul' );
+// TODO Internationalize this!
+const lastItemContent = `Search for pages containing [${ query }] ...`;
+
+// Process and append a Wikipedia Search API result page into an HTML list item.
+function appendResult( result, list ) {
+	const listItem = document.createElement( 'li' );
+
+	// Add optional thumbnail.
+	const thumbnailValue = result.thumbnail;
+	if ( thumbnailValue !== null ) {
+		const thumbnail = document.createElement( 'img' );
+		// The API doesn't return the URL protocol.
+		thumbnail.src = `https:${ result.thumbnail.url }`;
```

Comment:
```suggestion
		thumbnail.src = `https:${ thumbnailValue.url }`;
```
Avoid getting a property again.